scripts/report-ci-times.ps1

```diff
@@ -41,17 +51,27 @@ $priorities = @{
 
     Correctness_Determinism = 7;
     Correctness_Build = 7;
-    Correctness_SourceBuild = 7;
+    Correctness_TodoCheck = 7;
+    Correctness_Rebuild = 7;
+    Correctness_Analyzers = 7;
```

Comment:
we have both 'Correctness_Analyzers' and 'Correctness_Code_Analysis'?

Reply:
Well, so, Barbet added _Analyzers and that is in main. I also added the jobs from Jon's PR which adds _Code_Analysis but removes _Analyzers. Look we will need to update it again in the future.

Reply:
works for me. Yes I think it's good to have job names even if they are no longer in use since you don't know what historical range or branch the user is trying to measure.